com.zerobrane.studio.yml

```diff
@@ -1,33 +1,103 @@
 app-id: com.zerobrane.studio
 runtime: org.freedesktop.Platform
-runtime-version: '22.08'
+runtime-version: 23.08
```

Comment:
```suggestion
runtime-version: '23.08'
```

There is actually a flatpak-builder warning if you don't put these quotes.

Reply:
Maybe this should be an error, not a warning.

Reply:
I have never seen this breaking anything.

Reply:
It's still wrong. Branch is a string and you're writing it as a decimal number.

Reply:
> I have never seen this breaking anything.

That's because you ignore it. It appear when you build locally and it is in the build logs:

![image](https://github.com/flathub/com.zerobrane.studio/assets/114441/84dd49d8-bc41-47cc-8c27-255f5bd2c63d)

I missed the other occurence.

Reply:
I mean `flatpak-builder` still works and versions are numbers to me. Maybe print the warning into the GitHub messages if they are important to you.

Reply:
They are not numbers, they are **STRING** as per the schema.

yaml is just badly designed, because it tell from the philosophy of badly designed computer languages.

Reply:
Also a git tag is a **STRING**

From the documentation.

```
       runtime-version (string)
           The version of the runtime that the application uses, defaults to master.
```

```
           tag (string)
               The tag to use from the git repository
```

Reply:
I was more like comparing this to @chocolatey which is based on @nuget and it requires version numbers to follow https://semver.org so it can compare them automatically and I doubt they compare strings.